chains/evm/cli/bridge/set-burn.go

```diff
@@ -16,8 +16,8 @@ import (
 
 var setBurnCmd = &cobra.Command{
 	Use:   "set-burn",
-	Short: "Set a token contract as mintable/burnable",
-	Long:  "Set a token contract as mintable/burnable in a handler",
+	Short: "Set a token contract as burnable",
+	Long:  "Set a token contract as burnable by the handler contract",
```

Comment:
How about this line of code?
https://github.com/ChainSafe/chainbridge-solidity/blob/master/contracts/handlers/ERC20Handler.sol#L79
Does not it makes its mintable then as well?

Reply:
I guess it is the same net result, just seems strange to me since this is an indirect effect of making a token burnable; granting permission to burn allows for token to be minted since the token is within the `burnList` mapping.

I think we could make the description clearer. What do you think?

Reply:
What is here @waymobetta ?